### ISSUE TITLE
Incorrect version number in server error log if updated via web UI.

### ISSUE THREAD

**Issue by MailYouLater** [Ylianst/MeshCentral#745]
In another issue I posted an error log and everybody seemed surprised and/or disbelieving when I stated that the error had occurred after the server had been updated to a newer version than what it stated it was in the error log.  Well, I've found that with the settings MeshCentral initally defaults to, I can reliably crash the server by going into the server console and running `mpsstats`.  This is obviously a separate issue which should probably also be fixed, but for now this is an easy way of reliably reproducing the issue I'm reporting here, which is the fact that the error log shows the version that was initially started, not the version that is actively running.

In short, if you install MeshCentral via NPM, launch the server, update via the web UI, and crash the server, the error log will show the version number that you originally launched, not the version number that you updated to after launching it. See screen recording below (click for higher quality)

[![Demo](https://user-images.githubusercontent.com/1279475/71284068-bd46ae00-2316-11ea-94e6-97303aec9b9a.gif)](https://user-images.githubusercontent.com/1279475/71297739-8f269580-2339-11ea-9ae6-4c501219bd8b.gif)

**Comment by Ylianst** [comment]
!!! Wow, yep... Thank you for reporting this. Fixing both of these issues now :)

How did you make the animated video?

**Comment by Ylianst** [comment]
Just published MeshCentral v0.4.6-b with a fix for both the version string and "mpsstats". It was good timing, I was about to publish a new version and saw your post. Thanks for reporting.

**Comment by ryanblenis** [comment]
+1, also interested in what you used to make the video @MailYouLater

**Comment by MailYouLater** [comment]
To make the video I downloaded [fscPortable](https://portableapps.com/apps/graphics_pictures/fscportable), then manually updated the contents in the `App\fsc` folder with contents of the 'portable' `.zip` file at FastStone's download [FastStone Capture download page](https://www.faststone.org/FSCaptureDownload.htm). (The portable version uses the last truly free version, the newer version has new features, like video capture, but is shareware, so PortableApps.com won't use it.)  I used the video capture feature and the built in editor to export it as a `.gif`, then I had to run it through a couple of optimizers to get the size down to where it was acceptable to upload to GitHub.

**Comment by MailYouLater** [comment]
OK, it took me a little bit of extra work to be able to test this because you fixed `mpsstats` issue and I had to find a new way to crash the server.  Once I did find another way to crash the server, I was able to test it, and apparently if you initially started the server on a version older than `0.4.6-b`, then you'll still have this issue even after updating to newer versions.  Now that you've released version `0.4.6-c`, I've tested again, and if you've updated to `0.4.6-b`, completely restarted the server (not using the web UI) then update using the web UI, it will show the correct version.

**Comment by Ylianst** [comment]
Thanks for testing. In my case, I made myself a "crash" command... but I did not leave it in the version I published :)
